tests/torch/test_statistics_aggregator.py

```diff
@@ -62,7 +62,7 @@ def get_backend_model(self, dataset_samples):
 
     @pytest.fixture
     def is_backend_support_custom_estimators(self) -> bool:
```

Comment:
I see no reference in docs or code on how the word "estimator", or even a "custom estimator", is applicable to statistics colletion. What is the purpose of this flag, and why must it now be `True`?

Reply:
Ok, I discovered that the stat collector params can be deduced from `RangeEstimator` entities from POT world, but the notion of estimators lies above the level of stat collectors, and the unit tests for stat aggregators should not make references to it.

Reply:
I've added an explanation in docsting